test/Integration/gemm-loop-tiling-e2e-bf16.mlir

```diff
@@ -0,0 +1,132 @@
+// RUN: mlir-gen --kernel=args --batch=32 --layers=16,16 --tiles=4,4,4 -float-type=bf16 | tpp-run --M-tile-shape=2,4 --N-tile-shape=2,2 --loop-shuffle-order=0,2,1,3 --num-outer-parallel=2  -e=entry -entry-point-result=void -seed 123 -print | FileCheck %s
+// RUN: mlir-gen --kernel=args --batch=32 --layers=16,16 --tiles=4,4,4 -float-type=bf16 | tpp-run --def-parallel --parallel-task-grid=2,2  -e=entry -entry-point-result=void -seed 123 -print | FileCheck %s
+// CHECK:	( 0.0922852, 0.0820312, 0.114258, 0.149414 )
```

Comment:
This will probably not pass on AMD/Arm, need the same kind of regex as the other file below.

Reply:
The regex alone is not sufficient because in some cases, the differences could be something like 0.09 and 0.10, which I had to manually change in some cases.